package.json

```diff
@@ -10,20 +10,24 @@
     "prebuild": "rimraf dist",
     "validate": "npm-run-all --parallel lint cover build test:configs --sequential check-coverage",
     "build": "babel -d dist src",
-    "cover": "nyc --reporter=lcov --reporter=text --reporter=html npm run test",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "lint": "eslint .",
-    "test": "ava \"src/**/*.test.js\" --verbose --require ./other/setup-ava-env.js",
-    "test:scoped": "ava --verbose --require ./other/setup-ava-env.js",
-    "test:all": "npm-run-all --parallel test test:configs",
-    "test:configs": "ava tests/configs.js --verbose --require ./other/setup-ava-env.js",
-    "test:specific:config": "ava tests/specific-config.js --verbose --require ./other/setup-ava-env.js ",
-    "watch:test": "nodemon --quiet --watch src --exec npm run test -s",
-    "watch:test:scoped": "nodemon --quiet --watch src --exec npm run test:entry -s",
```

Comment:
:sweat_smile: yeeeeeah :-) It did for a while :-) With the new `--match` flag though, this wont be needed anyway :D